baseapp-auth/baseapp_auth/rest_framework/users/serializers.py

```
@@ -14,30 +18,51 @@
+    image = ThumbnailImageField(required=False, sizes={"small": (100,100)})
+
+    class Meta:
+        model = Profile
+        fields = ("id", "name", "image", "url_path")
+
+    def get_id(self, profile):
+        return get_obj_relay_id(profile)
+
+    def to_representation(self, profile):
+        data = super().to_representation(profile)
+        if (data['image'] != None):
```

Ruff (0.8.0)

Comparison to `None` should be `cond is not None`

Replace with `cond is not None`

(E711)